src/services/avert-services.ts

```diff
@@ -0,0 +1,33 @@
+import { AvertRecord } from "@/schema/avert";
+import { AvertModel } from "@/database/avert-model";
+import { Location, PowerPlantClass } from "@/schema/egrid";
+import { Error as CustomError } from "@/schema/error";
+
+export async function addAvertRecord(record: AvertRecord): Promise<void> {
+  try {
+    await AvertModel.create(record);
+  } catch (err) {
+    console.error("", err);
```

Comment:
This is good that we're logging the error, but another way to fix unused variable error is using the `catch` statement without a variable:

```
} catch {
...
}
```